### PR TITLE
Export getSimpleModelId instead

### PR DESCRIPTION
**Pull request by superkhau**
- Hide implementation details (such as Buffer and proquint)
- Export getSimpleModelId function instead

Connect to #67

### CODE REVIEW

**Comment by superkhau** [comment]
@bajtos Code review please.

File: client/lbclient/boot/replication.js [hunk 1]
```diff
@@ -51,4 +53,8 @@ module.exports = function(client) {
   });
 
   client.sync = sync;
+
+  client.getSimpleModelId = function(modelId) {
```
Comment (bajtos) [inline client/lbclient/boot/replication.js]: While you are at changing this, could you please use a more descriptive adjective? E.g. `getReadableModelId` or `getPronouncableId`.

**Comment by bajtos** [comment]
Much better :+1: 

See my one minor comment above, feel free to merge with or without addressing it.

File: client/lbclient/boot/replication.js [hunk 2]
```diff
@@ -51,4 +53,8 @@ module.exports = function(client) {
   });
 
   client.sync = sync;
+
+  client.getReadableModelId = function(modelId) {
```
Comment (superkhau) [inline client/lbclient/boot/replication.js]: @bajtos Renamed.

**Pull request merged**